### ISSUE TITLE
2.5.8 User Agent exception should be about the target with its spacing, not just the target

### ISSUE THREAD

**Issue by mitchellevan** [w3c/wcag#1682]
Currently [2.5.8](https://www.w3.org/TR/WCAG22/#pointer-target-spacing)  includes this User Agent exception:

> except when... the size of the target is controlled by the user agent and is not modified by the author

This would needlessly allow a web author to position small components close together, such as radio buttons, where the component itself is controlled by the user agent but the distance between components is not controlled by the user agent.

Proposed: 

> except when... the *size of the area that includes the target and no other targets* is controlled by the user agent and is not modified by the author

With this edit, the User Agent exception would still apply to some objects, such as browser-default spin button controls.

**Comment by alastc** [comment]
The latest version does not include the user-agent exception at all:
https://w3c.github.io/wcag/guidelines/22/#target-size-minimum

It has also been overhauled from a min-size requirement, so the group did not think the exception was required. Does that resolve / mitigate / side-step the issue for you?

**Comment by mitchellevan** [comment]
Thanks, I'm not sure how I missed it in the Editor's Draft. I'm closing this issue.
